AK/StringUtils.h

```diff
@@ -22,6 +22,11 @@ enum class TrimMode {
     Both
 };
 
+enum class TrimNumberString {
```

Comment:
A bit nit-picky, but `TrimNumberString` sounds kinda weird to me. Maybe just `TrimWhitespace`?

Reply:
`TrimWhitespace` doesn't explain _what_ it's trimming the whitespace from, however. And `TrimWhitespaceFromNumber` is a bit of a handful to type. :^)

Reply:
Doesn't really matter, the enum can be generic and later be used for other things - `string.to_int(TrimWhitespace::No)` seems pretty clear to me 🤔

Reply:
Yeah, I don't think there would be any confusion on what the whitespace is being trimmed from.